app/models/related_topic_list.rb

```diff
@@ -0,0 +1,38 @@
+# CLEANUP: 22/4 -  Related topics are currently in both the content-store and
```

Comment:
I'd skip the "22/4" here - we can use git to get the date (and that way we don't have to guess the data format)

I'd also say "FIXME" rather than "CLEANUP", since that's the more commonly used convention in our code (I think).